core/vibes/soul/primitives/product-card/index.tsx

```diff
@@ -149,6 +153,9 @@ export function ProductCard({
               </span>
             )}
             {price != null && <PriceLabel colorScheme={colorScheme} price={price} />}
+            {reviewsEnabled && typeof rating === 'number' && rating > 0 && (
```

Comment:
Elected to not render the `Rating` component if a product has no reviews. Otherwise, all product cards would contain 5 empty stars, which visually looks like they're all rated as 0.